src/operation/decision_model.jl

```diff
@@ -518,7 +518,7 @@ function write_model_dual_results!(
         if cols == get_time_steps(container)
             cols = ["System"]
         end
-        write_result!(store, STORE_CONTAINER_DUALS, key, timestamp, dual, cols)
+        write_result!(store, key, timestamp, dual, cols)
```

Comment:
I'd like to remove this from the interface. We can get it from the key.

Reply:
ok